submit_and_compare/static/html/submit_and_compare_edit.html

```diff
@@ -18,7 +18,15 @@
           </div>
           <span class="tip setting-help">This assigns an integer value representing the weight of this problem</span>
         </li>
-        
+
+        <li class="field comp-setting-entry is-set">
```

Comment:
It is possible, but I think we decided against it in order to allow Ross to pull changes from this repo without making the switch on the openlearningintiative repo